lab-geoff/test/test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+let Stack = require('../lib/stack.js');
+let expect = require('chai').expect;
+
+describe('stack module', function() {
+  describe('.push()function', function() {
+    it('should add a new node to the stack', function() {
+      let st = new Stack(5);
+      st.push(2);
+      expect(st.top.val).to.equal(2);
+    });
+    it('should increase in size when a new node is added', function() {
+      let st = new Stack(5);
+      let size = st.size;
+      st.push(5);
+      expect(st.size).to.equal(size += 1);
```

Comment:
This should be `size + 1` rather than `size +=1`, given that we don't want the += operator to actually mutate the value of size during the evaluation. It has the same effect at this point, but it's not actually testing the detail that you want.